.travis.yml

```diff
@@ -8,7 +8,6 @@ python:
     - 3.4
     - 3.5
     - "pypy"
-sudo: false
```

Comment:
Ah, I missed this change.

You use sudo bellow so even if  the sudo attribute is removed, it will still result in using legacy (non container) Travis CI infrastructure.

I think they added support for installing debian packages which still allows you to use new container based infrastructure. I will look into it.